localization/languages/pt-BR.json

```diff
@@ -31,43 +31,43 @@
         /* custom !bangs
         these are some of the items that show up when you press ! in the searchbar.
         Each one of these strings describes what the command will do when you run it. */
-        "showMoreBangs": null, //missing translation
+        "showMoreBangs": "Mostrar mais...",
         "viewSettings": "Ver configurações",
         "viewReadingList": "Ver listas de leitura",
         "takeScreenshot": "Capturar tela",
-        "clearHistory": "Limpar todo o histórico",
-        "clearHistoryConfirmation": null, //missing translation
+        "clearHistory": "Limpar histórico",
+        "clearHistoryConfirmation": "Limpar todo o histórico e dados de navegação?",
         "switchToTask": "Trocar para a tarefa",
-        "createTask": "Criar uma tarefa",
-        "closeTask": null, //missing translation
+        "createTask": "Criar tarefa",
+        "closeTask": "Fechar tarefa",
         "moveToTask": "Mover esta aba para uma tarefa",
-        "nameTask": null, //missing translation
-        "addBookmark": null, //missing translation
+        "nameTask": " Nomear tarefa",
+        "addBookmark": "Adicionar favorito",
 		"searchBookmarks": "Buscar nos favoritos",
-        "bookmarksAddTag": null, //missing translation
-		"bookmarksSimilarItems": null, //missing translation
-		"searchHistory": null, //missing translation
-        "importBookmarks": null, //missing translation
-		"exportBookmarks": null, //missing translation
+        "bookmarksAddTag": "Adicionar tag",
+		"bookmarksSimilarItems": "Itens semelhantes",
+		"searchHistory": "Buscar no histórico",
+        "importBookmarks": "Importar favoritos do arquivo HTML",
+		"exportBookmarks": "Exportar favoritos",
         /* navbar */
-        "openMenu": null, //missing translation
-        "enterReaderView": "Entrar no modo leitura",
+        "openMenu": "Menu",
+        "enterReaderView": "Modo leitura",
         "exitReaderView": "Sair do modo leitura",
         "emptyReadingListTitle": "Sua lista de leitura está vazia",
         "emptyReadingListSubtitle": "Artigos abertos no modo de leitura são listados aqui, e salvos offline por 30 dias.",
         "newTabLabel": "Nova aba", //this is a noun, used for tabs that don't have a page loaded in them yet
         "connectionNotSecure": "Sua conexão com este site não é segura.",
-        "searchbarPlaceholder": "Busque ou digite um endereço",
+        "searchbarPlaceholder": "Busque no %eng ou digite um endereço", //%eng corresponde ao nome do serviço de pesquisa
```

Comment:
Can you remove this for now so it can be reviewed separately as part of the other PR?